index.js

```diff
@@ -74,12 +82,12 @@ module.exports = {
     "array-callback-return": "error",
     "block-scoped-var": "error",
     "class-methods-use-this": "error",
-    complexity: ["error", 10],
+    "complexity": ["error", 10],
```

Comment:
keys that don't need `""` shouldn't have them

Reply:
I put quotes to the keys that weren't with quotes in order to make all rules with uniform and pretty syntax